pyrobuf/protobuf/templates/proto_pyx.tmpl

```diff
@@ -42,40 +80,42 @@ cdef class {{ name }}:
             components.append('}')
         return '\n'.join(components)
 
-    cpdef reset(self):
+    cpdef void reset(self):
```

Comment:
Does it make sense to combine `reset` and `Clear`? If not, should `reset` be made a "private" method?